### PR TITLE
add Nightly version to sidebar 

### PR DESCRIPTION
**Pull request by landylee007**
This change is [Reviewable](https://reviewable.io/reviews/koderover/zadig-doc/187)

### CODE REVIEW

File: dev/02.安装与配置/00.overview.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Zadig 正式版目前支持如下安装方式：
 
 ## Nightly 版
 
-Nightly 版本基于最新代码定时发布，适合快速尝鲜体验 Zadig 产品最新能力，不推荐生产使用。具体安装方式参见[`All in One 一键安装模式`](/dev/install/all-in-one/)
+Nightly 版本基于最新代码定时发布，适合快速尝鲜体验 Zadig 产品最新能力，不可生产使用。具体安装方式参见[`Nightly 版本`](/v1.7.1/install/nightly/)
```
Comment (fansi404) [inline dev/02.安装与配置/00.overview.md:21]: /dev/install/nightly

File: v1.7.1/02.安装与配置/01.all-in-one.md [hunk 2]
```diff
@@ -33,19 +33,25 @@ permalink: /v1.7.1/install/all-in-one/
...
-|---|---|---|
-|Nightly|官方下载源：[https://download.koderover.com/install?type=nightly](https://download.koderover.com/install?type=nightly)| 基于最新代码定时发布 |
-|正式版|官方下载源：[https://download.koderover.com/install?type=all-in-one](https://download.koderover.com/install?type=all-in-one) <br> GitHub 源：[https://github.com/koderover/zadig/releases/download/v1.7.1/all_in_one_install_quickstart.sh](https://github.com/koderover/zadig/releases/download/v1.7.1/all_in_one_install_quickstart.sh) | 适用于快速体验，不推荐生产使用 |
+根据网络情况，可从以下源选择相应的 Zadig 安装脚本并添加可执行权限。
+
+GitHub 源：
+
+```bash
+# 下载安装脚本并添加可执行权限
+curl -LO https://github.com/koderover/zadig/releases/download/dev/all_in_one_install_quickstart.sh
```
Comment (fansi404) [inline v1.7.1/02.安装与配置/01.all-in-one.md:42]: 404 not found

File: v1.7.1/02.安装与配置/04.Nightly.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
+---
+title: Nightly 版本
+date: 2021-12-16 16:48:02
+permalink: /dev/install/nightly/
```
Comment (fansi404) [inline v1.7.1/02.安装与配置/04.Nightly.md:4]: permalink: /v1.7.1/install/nightly

File: v1.7.1/02.安装与配置/04.Nightly.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
+---
+title: Nightly 版本
+date: 2021-12-16 16:48:02
+permalink: /1.7.1/install/nightly/
```
Comment (lilianzhu) [inline v1.7.1/02.安装与配置/04.Nightly.md:4]: permalink: /v1.7.1/install/nightly/

**Review by lilianzhu — APPROVED** [review]

**Pull request merged**